packages/typescript/src/index.ts

```diff
@@ -58,7 +58,11 @@ export default function typescript(options: RollupTypescriptOptions = {}): Plugi
     },
 
     buildEnd() {
-      if (process.env.ROLLUP_WATCH !== 'true') {
+      /** TODO: remove 'process.env.ROLLUP_WATCH' and only check for 'this.meta.watchMode' in the future
+       * 'this.meta.watchMode' is supported since rollup@2.14.0
+       * @see https://github.com/rollup/rollup/pull/3616
+       **/
+      if (process.env.ROLLUP_WATCH !== 'true' && this.meta.watchMode !== true) {
```

Comment:
If we're increasing the peer dependency, is there any reason to leave the env check?